src/content/contributing/building_testing/_index.en.md

```diff
@@ -45,29 +47,61 @@ To run Go unit tests:
 make test
 ```
 
-### End-to-end tests
+### Multi-Cluster KIND Based Environment {#clusters}
+
+Shipyard provides a basic target that creates a KIND based multi-cluster
+environment, without any special deployment (apart from the default K8s):
+
+```
+make clusters
+```
+
+Find out more about [Shipyard's clusters target](../shipyard#clusters).
+
+### Multi-Cluster Submariner Deployment
+
+Shipyard provides a basic target that deploys submariner on a KIND based
+multi-cluster environment (if one isn't deployed, this target will deploy it
+as well):
+
+```
+make deploy
+```
+
+Find out more about [Shipyard's deploy target](../shipyard#deploy).
```

Comment:
Same here: 

`Shipyard provides functionality to [deploy submariner](../shipyard#deploy) in a cluster environment.`